jsonnet/jsonnet.bzl

```diff
@@ -307,7 +356,9 @@ def _jsonnet_to_json_test_impl(ctx):
       list(transitive_data) +
       list(depinfo.transitive_sources) +
       [list(jfile.files)[0] for jfile in jsonnet_ext_str_files] +
-      [list(jfile.files)[0] for jfile in jsonnet_ext_code_files])
+      [list(jfile.files)[0] for jfile in jsonnet_ext_code_files] +
```

Comment:
Feel free to ignore the comment (issue was already present), but this looks suspicious: `list(jfile.files)[0]`
(is it possible to have multiple files in the list?)

Reply:
Yeah that was my code and when I wrote this originally I was not aware of idiomatic usage. This PR https://github.com/bazelbuild/rules_jsonnet/pull/67 should fix that though. If you also have the time to look over that one that would be very much appreciated.